### DISCUSSION TITLE
http.timeout in milliseconds or allow float

### DISCUSSION THREAD

**Discussion by sergejostir** [drakkan/sftpgo#427] [Ideas]
Hello.

Is there a specific reason why http.timeout config option only accepts integers which are considered seconds? My proposal is to either accept integer and interpret it as milliseconds or allow setting this config value to a float (so we can set for example 3.5 seconds).

**Comment by drakkan** [comment]
Hi,

thank you for your suggestion. To avoid a backward incompatible change maybe we could accept float values. Does 3.5 seconds make a big difference to your use case compared to 3 or 4?

**Reply by sergejostir** [reply]
Well not really. I wanted to set timeout 250ms higher than my gateway, so the sftpgo's client won't timeout before the gateway does, but it will also do just fine with 1 second higher.

**Reply by drakkan** [reply]
It is supported now https://github.com/drakkan/sftpgo/commit/8ecf64f4813deb6bb5eb6d17ebc46344ad406423 please confirm that it works as expected, thank you
